packages/wallet/src/index.ts

```diff
@@ -1,2 +1,2 @@
-export * from './wallet';
-export * from './provider';
+export * as transfer from './request-transfer';
```

Comment:
this is a breaking change. Don't do something like that in SDKs. You need to have a very good reason for that